### PR TITLE
use aladin for skymap on target list page

### PR DESCRIPTION
**Pull request by sglasberg**
aladin is now on the target list page

### CODE REVIEW

**Comment by sglasberg** [comment]
@jchate6 I dont know why these tests are failing please take a look!

File: tom_targets/templates/tom_targets/partials/aladin_skymap.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!-- insert this snippet where you want Aladin Lite viewer to appear -->
```
Comment (jchate6) [inline tom_targets/templates/tom_targets/partials/aladin_skymap.html:1]: Remove this line, but maybe replace it with a comment about where found the example code?

File: tom_targets/templatetags/targets_extras.py [hunk 2]
```diff
@@ -303,15 +255,40 @@ def select_target_js():
...
     and a scale bar. The resulting image is downloadable. This templatetag only works for sidereal targets.
     """
+
     return {'target': target}
 
 
+@register.inclusion_tag('tom_targets/partials/aladin_skymap.html')
+def aladin_skymap(targets):
+    """
+    IN PROGRESS: should display aladin skyview on Target Distribution skymap
```
Comment (jchate6) [inline tom_targets/templatetags/targets_extras.py:271]: Update these comments to represent what this function does now that it's no longer in progress.
Consider including links to Aladin Docs.

File: tom_targets/templatetags/targets_extras.py [hunk 3]
```diff
@@ -303,15 +255,40 @@ def select_target_js():
...
+            ra = target.ra
+            dec = target.dec
+            target_list.append({'name': name, 'ra': ra, 'dec': dec})
+
+    context = {'targets': target_list}
+    return context
+
+
+@register.inclusion_tag('tom_targets/partials/aladin_skymap.html')
+def target_distribution(targets):
```
Comment (jchate6) [inline tom_targets/templatetags/targets_extras.py:288]: Add a doc string describing why you included this function.

**Review by jchate6 — CHANGES_REQUESTED** [review]
A few requests for comments.

**Review by jchate6 — APPROVED** [review]

**Pull request merged**